easybuild/easyconfigs/e/EvidentialGene/EvidentialGene-2018.01.01-intel-2017a-Perl-5.24.1.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = "PackedBinary"
+
+name = "EvidentialGene"
+version = '2018.01.01'
+
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://arthropods.eugenes.org/EvidentialGene/'
+description = """EvidentialGene is a genome informatics project for 
+ "Evidence Directed Gene Construction for Eukaryotes",
+ for constructing high quality, accurate gene sets for
+ animals and plants (any eukaryotes), being developed by
+ Don Gilbert at Indiana University, gilbertd at indiana edu."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['ftp://arthropods.eugenes.org/evigene_old/']
+month = ['', 'jan', 'feb', 'mar', 'apr', 'may', 'jun', 'jul', 'aug', 'sep', 'oct', 'nov', 'dec']
+dlver = version.split('.')[0][-2:]
+dlver += month[int(version.split('.')[1])]
+dlver += version.split('.')[2]
+sources = ['evigene%s.tar' % dlver]
+checksums = ['6972108112cdb1fb106da11321d06b09f518b544e4739d0bf19da1984131e221']
+
+dependencies = [
+    ('Perl', '5.24.1', '', ('GCCcore', '6.3.0')),
```

Comment:
Why hardcode to `GCCcore/6.3.0` subtoolchain, there's no need to?

Reply:
wanted to be sure that the `GCCcore` version is used. (we have `Intel` version as well)